syntax = "proto3";

package capsule;

import "capsule/common.proto";

message CrawlRequest {
    repeated bytes sids = 1;
    repeated string urls = 2;
}

service Worker {
    rpc CrawlUrl(CrawlRequest) returns (stream Content);
}
